Classes/FontAwesome.swift

```diff
@@ -659,3 +659,23 @@ public extension String {
     return name.rawValue.substringToIndex(advance(name.rawValue.startIndex, 1))
   }
 }
+
+public extension UIImage {
+    public static func fontAwesomeIconWithName(name: FontAwesome, textColor: UIColor, size: CGSize) -> UIImage {
+        let paragraph = NSMutableParagraphStyle()
+        paragraph.lineBreakMode = NSLineBreakMode.ByWordWrapping
+        paragraph.alignment = .Center
+        let attributedString = NSAttributedString(string: String.fontAwesomeIconWithName(name) as String, attributes: [NSFontAttributeName: UIFont.fontAwesomeOfSize(24.0), NSForegroundColorAttributeName: textColor, NSParagraphStyleAttributeName:paragraph])
```

Comment:
Reasonable size for usage in UITabBarItem, feel free to add another parameter in fontAwesomeIconWithName(…).
> Am 06.07.2015 um 09:59 schrieb UglyTroLL <notifications@github.com>:
> 
> In Classes/FontAwesome.swift <https://github.com/thii/FontAwesome.swift/pull/15#discussion_r33912480>:
> 
> > @@ -659,3 +659,23 @@ public extension String {
> >      return name.rawValue.substringToIndex(advance(name.rawValue.startIndex, 1))
> >    }
> >  }
> > +
> > +public extension UIImage {
> > +    public static func fontAwesomeIconWithName(name: FontAwesome, textColor: UIColor, size: CGSize) -> UIImage {
> > +        let paragraph = NSMutableParagraphStyle()
> > +        paragraph.lineBreakMode = NSLineBreakMode.ByWordWrapping
> > +        paragraph.alignment = .Center
> > +        let attributedString = NSAttributedString(string: String.fontAwesomeIconWithName(name) as String, attributes: [NSFontAttributeName: UIFont.fontAwesomeOfSize(24.0), NSForegroundColorAttributeName: textColor, NSParagraphStyleAttributeName:paragraph])
> Why hardcoded the font size to 24?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/thii/FontAwesome.swift/pull/15/files#r33912480>.
>